packages/desktop-client/src/components/modals/SimpleFinInitialise.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+
+import { send } from 'loot-core/src/platform/client/fetch';
+
+import { Error } from '../alerts';
+import { ButtonWithLoading } from '../common/Button';
+import ExternalLink from '../common/ExternalLink';
```

Comment:
Could this be the cause of your build errors?  Default exports were recently disallowed in lint

Reply:
Yes! I thought I had cleared that up in the merge conflicts but apparently there were more. Thanks!